### DISCUSSION TITLE
In my notes in markdown how should I write the links to images?

### DISCUSSION THREAD

**Discussion by Julianoe** [binyamin/eleventy-garden#90] [Q&A]
I have /notes/category/mynote.md as a file structure. When I write a note, all images for all notes are in the same category folder the notes are in. But I can't figure out a way for it to work. It seems it never pulls out the images to display them in the notes pages. Do you know of a working implementation similar to this?

**Comment by Julianoe** [comment]
For a given image that would give it and url like my11tygarden.com/notes/category/myimage.jpg

In the mean time I'll just pull all images to a /images folder but that's not ideal for me.

**Comment by binyamin** [comment]
Sorry for the wait. This is really an eleventy question. You could ask in the 11ty discord.
